### ISSUE TITLE
Doesn't work with data with a single b0

### ISSUE THREAD

**Issue by arokem** [dPys/EddyMotionCorrection#3]
I get this kind of error for a dataset that has only one b0 (https://openneuro.org/datasets/ds000206/versions/1.0.0):

```
Node: emc_correction.emc_wf.split_b0s_node
Working directory: /tmp/emc_correction/emc_wf/split_b0s_node

Node inputs:

function_str = def save_4d_to_3d(in_file):
    files_3d = nb.four_to_three(nb.load(in_file))
    out_files = []
    for i, file_3d in enumerate(files_3d):
        out_file = fname_presuffix(in_file, suffix="_tmp_{}".format(i))
        file_3d.to_filename(out_file)
        out_files.append(out_file)
    del files_3d
    return out_files

in_file = /tmp/emc_correction/emc_wf/extract_b0_node/sub-THP0001_ses-THP0001UW1_acq-GD33_run-01_dwi_b0.nii.gz

Traceback (most recent call last):
  File "/usr/local/miniconda/lib/python3.7/site-packages/nipype/pipeline/plugins/multiproc.py", line 67, in run_node
    result["result"] = node.run(updatehash=updatehash)
  File "/usr/local/miniconda/lib/python3.7/site-packages/nipype/pipeline/engine/nodes.py", line 516, in run
    result = self._run_interface(execute=True)
  File "/usr/local/miniconda/lib/python3.7/site-packages/nipype/pipeline/engine/nodes.py", line 635, in _run_interface
    return self._run_command(execute)
  File "/usr/local/miniconda/lib/python3.7/site-packages/nipype/pipeline/engine/nodes.py", line 741, in _run_command
    result = self._interface.run(cwd=outdir)
  File "/usr/local/miniconda/lib/python3.7/site-packages/nipype/interfaces/base/core.py", line 434, in run
    runtime = self._run_interface(runtime)
  File "/usr/local/miniconda/lib/python3.7/site-packages/nipype/interfaces/utility/wrappers.py", line 142, in _run_interface
    out = function_handle(**args)
  File "<string>", line 2, in save_4d_to_3d
  File "/usr/local/miniconda/lib/python3.7/site-packages/nibabel/funcs.py", line 173, in four_to_three
    raise ValueError('Expecting four dimensions')
ValueError: Expecting four dimensions
```

**Comment by arokem** [comment]
Might come from here: https://github.com/dPys/EddyMotionCorrection/blob/main/emc/workflows/correct.py#L638

**Comment by dPys** [comment]
yep, the most obvious use-case I hadn't yet considered :-p 

I'll get to this shortly.

**Comment by dPys** [comment]
Resolved with #7 . Closing!
